CHANGELOG.md

```diff
@@ -1,3 +1,141 @@
+### citus v10.0 (February 16, 2021) ###
+
+* Adds support for per-table option for columnar storage
+
+* Adds `rebalance_table_shards` to rebalance the shards across the nodes
+
+* Adds `citus_drain_node` to move all shards away from any node
+
+* Enables single-node Citus for production workloads
+
+* Adds support for local distributed/subquery joins
+
+* Enables foreign keys between reference tables and local tables
+  (`citus.enable_local_reference_table_foreign_keys`)
+
+* Adds support for co-located/recurring correlated subqueries
+
+* Adds `alter_distributed_table` and `alter_table_set_access_method` UDFs
+
+* Adds `alter_old_partitions_set_access_method` UDF to compress old partitions
+
+* Adds cascade option to `undistribute_table` UDF for foreign key connected
+  relations
+
+* Allows distributed table creation immediately after CREATE EXTENSION citus
+
+* Automatically adds coordinator to the metadata before the first distributed
+  table created
+
+* Introduces adaptive connection management for local nodes
```

Comment:
Minor: can we get local node related things close to each other, most of them are already here, just move few of the others here:
- Allows distributed table creation immediately after CREATE EXTENSION citus
- Automatically adds coordinator to the metadata before the first distributed table created
- Introduces adaptive connection management for local nodes
- Adds support for local execution in INSERT..SELECT commands that perform re-partitioning
- Adds support for local execution in INSERT..SELECT commands that perform re-partitionin

Reply:
c2aaed64db4937655de8ab7ee6cc317ec490ab4c